docs/products/kafka/kafka-connect/howto/gcs-sink.rst

```diff
@@ -0,0 +1,129 @@
+Create a Google Cloud Storage sink connector
+============================================
+
+The Apache Kafka Connect® Google Cloud Storage (GCS) sink connector by Aiven enables you to move data from an Aiven for Apache Kafka® cluster to a Google Cloud Storage bucket for long term storage. The full connector documentation is available in the dedicated `GitHub repository <https://github.com/aiven/aiven-kafka-connect-gcs>`_.
+
+
+
+Prerequisites
+-------------
+
+To setup the Google Cloud Storage sink connector by Aiven, you need an Aiven for Apache Kafka® service :doc:`with Apache Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`.
+
+Furthermore you need to follow the steps :doc:`to prepare the GCP account and GCS sink <gcs-sink-prereq>` and collect the following information about the target GCS bucket upfront:
+
+* ``GCS_NAME``: The name of the GCS bucket
+* ``GCS_CREDENTIALS``: The Google service account JSON service key :ref:`created during the prerequisit phase <gcs-sink-connector-google-account>`
```

Comment:
typo: "prerequisit" missing the "e". (spotted by @kevinbowman-aiven earlier). 
sorry @ftisiot, but I think that's the last one!

Reply:
Fixed, thanks